modules/sandboxed-containers-deleting-namespace-web-console.adoc

```diff
@@ -4,24 +4,24 @@
 
 :_content-type: PROCEDURE
 [id="sandboxed-containers-deleting-namespace-web-console_{context}"]
-= Deleting a namespace using the web console
+= Deleting {sandboxed-containers-first} namespace
 
-You can delete a namespace by using the {product-title} web console.
+After you run the preceding commands, your cluster is restored to the state that it was prior to the installation process. You can now revoke namespace access to the Operator by deleting the `openshift-sandboxed-containers-operator` namespace.
 
 .Prerequisites
 
-* You have {product-title} {product-version} installed on your cluster.
+* You have {product-title} 4.10 installed on your cluster.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
 .Procedure
 
 . Navigate to *Administration* → *Namespaces*.
-. Locate the `openshift-sandboxed-containers-operator` namespace to delete in the list of namespaces.
-. On the rightmost side of the namespace listing, select *Delete Namespace* from the *Options* menu .
-. When the *Delete Namespace* pane opens, enter `openshift-sandboxed-containers-operator` in the field.
+. Search for the `openshift-sandboxed-containers-operator` namespace.
```

Comment:
```suggestion
. Search for the `openshift-sandboxed-containers-operator` namespace in the drop-down list of namespaces.
```

Reply:
I thought I'd deleted "in the drop-down list of namespaces". Checking now.